File: AMS.Web/src/app/models/batch-allocation/search-batch-matching-assessor-request-params.ts
export class SearchBatchMatchingAssessorRequestParams 
{
    constructor(){}

    BatchId : string = null;
    BatchName? : string = null;
    AssessorName? : string = null;
    AssessmentDate:Date = null;
    AssessmentTiming: boolean = null ; //Batch Assessment timing like "AM or PM" only
}

File: AMS.API/DTO/BatchAllocation/SearchBatchMatchingAssessorRequestParams.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace AMS.API.DTO.BatchAllocation
{
    public class SearchBatchMatchingAssessorRequestParams
    {
        public string BatchId { get; set; }

        public string BatchName { get; set; }

        public string AssessorName { get; set; }

        public DateTime AssessmentDate { get; set; }

        public bool AssessmentTiming { get; set; } //Batch Assessment timing like "AM or PM" only
    }
}